src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/databaseFeedManager.hpp

```diff
@@ -30,6 +31,7 @@
 
 constexpr auto DESCRIPTION_DATABASE_PATH {"queue/vd/descriptions"};
 constexpr auto REMEDIATIONS_DATABASE_PATH {"queue/vd/remediations"};
+constexpr auto ROCKS_DB_PATH {"./test.db"};
```

Comment:
Will this database used in production?

Reply:
I'm not sure it was not specified in this issue, but I can change it to use the same path used above.